app-misc/egads/egads-0.9.5-r2.ebuild

```diff
@@ -15,7 +15,13 @@ KEYWORDS="~amd64 ~ppc x86"
 
 EGADS_DATADIR=/var/run/egads
 
+PATCHES=(
+	"${FILESDIR}"/egads-0.9.5-make-build-work-with-clang16.patch
+)
+
 src_prepare() {
+	aclocal || die
+	eautoreconf
```

Comment:
btw: default should be before eautoreconf, so that users can apply patches which touch configure.ac. I'll fix these nits though.